draft-ietf-quic-tls.md

```diff
@@ -690,23 +690,51 @@ Client SHOULD NOT reuse tickets as that allows entities other than the server
 to correlate connections; see Section C.4 of {{!TLS13}}.
 
 
-## Enabling 0-RTT {#enable-0rtt}
+## 0-RTT
+
+The 0-RTT feature in QUIC allows a client to send application data before the
```

Comment:
s/The 0-RTT feature/0-RTT handshake delay

Reply:
Sorry, I don't know where "delay" comes into this.